sydent/config/sms.py

```diff
@@ -11,12 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import configparser
 from configparser import ConfigParser
 from typing import Dict, List
 
+import sydent.sms.openmarket
 from sydent.config._base import BaseConfig
 from sydent.config.exceptions import ConfigError
+from pydoc import locate
```

Comment:
This seems to be adding a dependency on pydoc? I think synapse has a way of doing this with no dependencies.